src/api/java/mezz/jei/api/recipe/category/IRecipeCategory.java

```diff
@@ -36,9 +37,22 @@
 	/**
 	 * Returns the localized name for this recipe type.
 	 * Drawn at the top of the recipe GUI pages for this category.
+	 * @deprecated Prefer calling and using {@link #getTitleAsTextComponent()}.
 	 */
+	@Deprecated
 	String getTitle();
```

Comment:
Should this be defaulted to:
```java
return getTitleAsTextComponent().getString();
```
I wasn't sure how clear it would be if it was that one of the get titles needs to be implemented, which is why I didn't do so for now.

Reply:
What you have now is the right way, we shouldn't give this a default. Unfortunately new methods like `getTitleAsTextComponent` will always need a default implementation for compatibility, and it is confusing if they are both defaulted and point at each other.